cmd/dfget/cmd/root.go

```diff
@@ -160,9 +160,9 @@ func runDfget() error {
 	daemonClient, err := checkAndSpawnDaemon()
 	if err != nil {
 		logger.Errorf("check and spawn daemon error:%v", err)
```

Comment:
When err != nil, why not return err? @Louhwz

Reply:
if err == nil , from the source to download

Reply:
oh. Study. Thanks.

Reply:
@yxxhero you can close the suggestion below now